src/components/SurveyDateForm.tsx

```diff
@@ -23,7 +24,8 @@ const SurveyDateForm = ({ onSubmitFunction }: Props): ReactElement => {
         {
             name: "Date",
             type: "date",
-            initial_value: dateFormatter(new Date()).format("YYYY-MM-DD"),
+            initial_value: dateFormatter(new Date()).format(mirDateFormat("-"))
```

Comment:
I think this is changing the date format - i.e. `DD-MM-YYYY` instead of `YYYY-MM-DD`?

Reply:
I also wonder it if it's worth having two different data format functions, one for humans (`DD/MM/YYYY`) and one for computers (`YYYY-MM-DD`) with descriptive names?